src/current/cockroachcloud/network-authorization.md

```diff
@@ -1,32 +1,29 @@
 ---
 title: Network Authorization for CockroachDB Cloud Clusters
-summary: Learn about the network authorization features for {{ site.data.products.db }} clusters.
+summary: Learn about the network authorization features for CockroachDB Cloud clusters.
 toc: true
 docs_area: manage
 ---
 
-{{ site.data.products.dedicated }} allows explicit configurations of the networks that can access your cluster.
-
 To prevent denial-of-service attacks, brute force password attacks, and other forms of malicious activity, Cockroach Labs recommends restricting your network to allow access only from specific IP address ranges controlled by your organization. These might include specific networks for your application deployments, hardened administrator access points, or disaster recovery pipelines.
 
+This page describes these options and how they help to protect {{ site.data.products.dedicated }} and {{ site.data.products.serverless }} clusters.
+
 ## Options for controlling network access
 
 You can authorize network access to your cluster by:
 
 - [Adding an authorized range of public IP addresses](#ip-allowlisting).
-- Setting up [Google Cloud Platform (GCP) Virtual Private Cloud (VPC) peering](#vpc-peering) or [Amazon Web Service (AWS) PrivateLink](#aws-privatelink) for your cluster (Dedicated clusters only). Access via [GCP VPC peering](#vpc-peering) or [AWS PrivateLink](#aws-privatelink) avoids traversing the public network, and therefore offers several advantages:
+- Setting up private connectivity so that inbound connections to your cluster from your cloud tenant are made the cloud provider's private network rather than the public internet, for enhanced network security and reduced network latency. If you use IP allowlisting rules together with private connectivity, private networks do not need to be added to that allowlist.
```

Comment:
```suggestion
- Setting up private connectivity so that inbound connections to your cluster from your cloud tenant are made over the cloud provider's private network rather than over the public internet, for enhanced network security and reduced network latency. If you use IP allowlisting rules together with private connectivity, private networks do not need to be added to that allowlist.
```